<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uARM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="ghg" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>761</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QSlider" name="ymover">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>250</y>
      <width>31</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="xmover">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>360</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="sliderPosition">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberX">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>410</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberY">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>410</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SPort">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetZero">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>40</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Set zero</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Enabled_B">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Disabled_B">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>240</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="UP">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>80</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DOWN">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>130</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RIGHT">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>170</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LEFT">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FRONT">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>110</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BACK">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>110</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="feedback">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Feedback</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Y">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>480</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;12&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Z">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>520</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;12&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>490</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="beta">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>510</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="alpha">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>470</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X in cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>520</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z in cm</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="m">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>480</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>alpha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>520</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Beta</string>
    </property>
   </widget>
   <widget class="QLabel" name="mlabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="gamma">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gamma_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>260</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gamma</string>
    </property>
   </widget>
   <widget class="QLabel" name="gamma_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>300</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>phi</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="phi">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="X">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>440</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>490</y>
      <width>41</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Y in cm</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="theta">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>550</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>560</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Theta</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="s">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mlabel_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>190</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readfile">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadFile</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="espera">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="gamma_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>350</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>espera(ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>570</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrist</string>
    </property>
   </widget>
   <widget class="QDial" name="wrist">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>590</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="grip">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>590</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>570</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Grip</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1216</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
